lib/tasks/order_cycles.rake

```diff
@@ -0,0 +1,19 @@
+namespace :ofn do
+  desc 'destroys the specified order cycle'
+  task :remove_order_cycle, [:order_cycle_id] => :environment do |_task, args|
+    remove_order_cycle(args.order_cycle_id)
+  end
+
+  desc 'destroys all order cycles of the specified coordinator'
+  task :remove_coordinated_order_cycles, [:coordinator_id] => :environment do |_task, args|
```

Comment:
I think that we should supply id and name to add an additional layer of safety.

Reply:
I can ask for the coordinator's permalink, as you suggested in another PR.